src/ring_theory/adjoin_root.lean

```diff
@@ -369,6 +369,79 @@ by rw [minpoly_power_basis_gen hf', hf.leading_coeff, inv_one, C.map_one, mul_on
 
 end power_basis
 
+section minpoly
+
+variables [comm_ring R] [comm_ring S] [algebra R S] (x : S) (R)
+
+open algebra polynomial
+
+/-- The algebra morphism `adjoin_root (minpoly R x) →ₐ[R] adjoin R x` -/
```

Comment:
Maybe it's clearer what this does when we try to approximate mathematical notation?
```suggestion
/-- The surjective algebra morphism `R[X]/(minpoly R x) → R[x]`.

If `R` is a GCD domain and `x` is integral, this is an isomorphism `adjoin_root.minpoly.equiv_adjoin`. -/
```